packages/remediations/src/NewRemediationWizard.js

```diff
@@ -138,63 +91,109 @@ class RemediationWizard extends Component {
+            .then(({ id }) => resolver(id, formValues[SELECT_PLAYBOOK], true));
         }
-    }
+    };
 
-    render () {
-        return (
-            this.state.open ?
-                <FormRenderer
-                    schema={this.state.schema}
-                    container={this.container}
-                    subscription={{ values: true }}
-                    FormTemplate={this.getFormTemplate}
-                    initialValues={{
-                        [HAS_MULTIPLES]: !!this.state.resolutions?.find(r => r.resolutions.length > 1),
-                        [MANUAL_RESOLUTION]: true,
-                        [SELECTED_RESOLUTIONS]: {},
-                        [EXISTING_PLAYBOOK_SELECTED]: false
-                    }}
-                    componentMapper={{
-                        [componentTypes.WIZARD]: Wizard,
-                        ...this.state.mapperExtension
-                    }}
-                    onSubmit={(_, formOptions) => {
-                        this.onSubmit(formOptions.getState().values);
-                        this.setOpen(false);
-                    }}
-                    onCancel={this.closeWizard}
-                />
-                : <Fragment/>
-        );
-    }
-}
+    const mapperExtension = {
+        'select-playbook': {
+            component: state.errors.length > 0 ? FetchError : SelectPlaybook,
+            issues: data.issues,
+            systems: data.systems
+        },
+        'review-actions': {
+            component: ReviewActions,
+            issues: data.issues,
+            issuesMultiple: state.issuesMultiple
+        },
+        'issue-resolution': {
+            component: IssueResolution,
+            systems: data.systems,
+            resolutions: state.resolutions
+        },
+        review: {
+            component: Review,
+            data: data,
+            issuesById: state.issuesById,
+            resolutions: state.resolutions
+        }
+    };
+
+    return (
+        state.isOpen ?
+            <FormRenderer
+                schema={state.schema}
+                subscription={{ values: true }}
+                FormTemplate={(props) => <Pf4FormTemplate {...props} showFormControls={false} />}
+                initialValues={{
+                    [HAS_MULTIPLES]: !!state.resolutions?.find(r => r.resolutions.length > 1),
+                    [MANUAL_RESOLUTION]: true,
+                    [SELECTED_RESOLUTIONS]: {},
+                    [EXISTING_PLAYBOOK_SELECTED]: false
+                }}
+                componentMapper={{
+                    [componentTypes.WIZARD]: Wizard,
+                    ...mapperExtension
+                }}
+                onSubmit={(_, formOptions) => {
+                    onSubmit(formOptions.getState().values);
+                    closeWizard();
+                }}
+                onCancel={closeWizard}
+            />
+            : <Fragment/>
+    );
```

Comment:
Showing an empty fragment is unnecessary. You can use `null`.

You can also optimize it a little if you move the condition right after the hook.

```jsx
useEfffect(...);

if(!isOpen) { 
  return null;
}
```

So the code won't go over the mapper and functions definitions.